packages/ag-charts-website/src/content/docs/licensing/index.mdoc

```diff
@@ -23,7 +23,7 @@ Please email [info@ag-grid.com](mailto:info@ag-grid.com) to receive a trial lice
 
 ### Installing AG Charts Enterprise
 
-The [Installation](installation) page contains instructions on how to import AG Charts Enterprise and set your license key.
+The [Installation](./installation/) page contains instructions on how to import AG Charts Enterprise and set your license key.
```

Comment:
```suggestion
The [Installation](/installation/) page contains instructions on how to import AG Charts Enterprise and set your license key.
```

Reply:
It's working on staging.
There seems to be differences between behaviour on local and staging.